### PR TITLE
Adds a load balancer

### PR DESCRIPTION
**Pull request by DRBeauchamp**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -1,3 +1,8 @@
 variable "region" {
   default = "us-east-1"
 }
+
+variable "certificate_arn" {
+  default     = "arn:aws:acm:us-east-1:259677325264:certificate/3ca66a19-4cbb-48b4-917f-ebdf2ec34b53"
```
Comment (jmound) [inline variables.tf:6]: @DRBeauchamp One way to not hard code an ARN here would be to use this: https://www.terraform.io/docs/providers/aws/d/acm_certificate.html

**Comment by jmound** [comment]
👍

**Comment by DRBeauchamp** [comment]
Ah yes. I am stuck on almost this exact thing on my route53 branch, where
for some reason, the following code in route53.tf isn't working.

```
# Get my Route 53 hosted zone id:
data "aws_route53_zone" "davops" {
   name         = "davops.net."
   *zone_id      = "Z047957726H4FRZU74N0Z"*  *### This is the number I'm
after, just being overly verbose here while I was trying to get it to work*
   private_zone = false
 }

# Create new Alias A-record set in that hosted zone:
resource "aws_route53_record" "web" {
  zone_id = data.aws_route53_zone.davops.zone_id
  name    = "web.davops.net."
  type    = "A"

  alias {
    name                   = aws_lb.load-balancer.dns_name
    zone_id                = aws_lb.load-balancer.zone_id
    evaluate_target_health = true
  }
}
```

It gives back:

Error: Reference to undeclared resource
  on route53.tf line 12, in output "zone_id":
  12:   value = aws_route53_zone.davops.zone_id
A managed resource "aws_route53_zone" "davops" has not been declared in the
root module.


How is this not considered declaring it? Is this not part of the root
module? I tried moving that part to main.tf

**Comment by DRBeauchamp** [comment]
Forget it. I got it to work. I just had a bad combination of arguments in
the data declaration block. Turns out you can be too verbose.

**Comment by jmound** [comment]
You're killin'  it

**Pull request merged**